### ISSUE TITLE
[Adaptive Cards> Close button]: Tooltip is not defined for the 'Close' button which is present on left pane navigation controls.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#7786]
### Target Platforms

UWP

### SDK Version

3.0.2206.2001

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS version: 22H2 (Build 25182.1010)
Browser: Edge Dev (106.0.1356.0)
Application: AdaptiveCardVisualizer
Application Version: 3.0.2206.2001

**Repro Steps:**

1. Open the Adaptive Card Visualizer application.
2. Navigate to the 'Close' button present on left pane navigation.
3. Observe the issue.

**Actual Result:**
For 'Close' button on the left pane navigation, tooltip is not defined.

**Expected Result:**
For 'Close' button, proper tooltip should be defined.

**User Impact:**
Person with cognitive disability will get impacted if tooltip is not defined due to which user will not be able to interact with the content or will have difficulty in understanding the purpose of the content properly.

**MAS Reference link:**
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.03.03

**“Have feedback to share on Bugs?** Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-vaibgu%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400980259288%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=TrcBaCyAPKg0WAQfdSZJP2i9pghMEDg59i2NeUUs5EQ%3D&reserved=0) “

### Screenshots

Uploading MAS1.3.3 Tooltip is not defined for 'Close' button.mp4…

### Card JSON

```JSON
NA
```
